ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -257,119 +275,228 @@ const ConfirmPaymentMethodScreen: React.FC<Props> = (props: Props) => {
+              mainText={paymentMethodCaption}
+              subText={
+                paymentMethodSubject +
+                (!isPayingWithPaypal ? ` · ${paymentMethodExpiration}` : "")
+              }
+              ctaText={I18n.t("wallet.ConfirmPayment.edit")}
+              onPress={props.pickPaymentMethod}
+              accessibilityLabel={`${paymentMethodA11yLabel}, ${I18n.t(
+                "wallet.ConfirmPayment.accessibility.edit"
+              )}`}
             />
-            {/* show the ability to change psp only when the payment method is a credit card */}
-            {!isPayingWithPaypal && (
-              <>
-                <View spacer={true} />
-                {maybePsp.isNone() ? (
-                  <H4 weight={"Regular"}>{I18n.t("payment.noPsp")}</H4>
-                ) : (
-                  <H4 weight={"Regular"}>
-                    {I18n.t("payment.currentPsp")}
-                    <H4>{` ${maybePsp.value.businessName}`}</H4>
-                  </H4>
-                )}
-                <Link onPress={props.pickPsp} weight={"Bold"}>
-                  {I18n.t("payment.changePsp")}
-                </Link>
-                <View spacer={true} large={true} />
-
-                {showFeeContextualHelp && (
-                  <Link onPress={showHelp} testID="why-a-fee">
-                    {I18n.t("wallet.whyAFee.title")}
-                  </Link>
-                )}
-              </>
-            )}
-            {isPayingWithPaypal && (
+
+            <View spacer large />
+
+            <View style={styles.iconRow}>
+              <TagIcon width={20} height={20} />
+
+              <H3
+                color="bluegrey"
+                style={styles.iconRowText}
+                accessibilityRole="header"
+              >
+                {I18n.t("wallet.ConfirmPayment.transactionCosts")}
+              </H3>
+            </View>
+
+            <View spacer />
+
+            <SelectionBox
+              mainText={formattedFees}
+              subText={pspName}
+              ctaText={
+                canChangePsp ? I18n.t("wallet.ConfirmPayment.edit") : undefined
+              }
+              onPress={canChangePsp ? handleChangePsp : undefined}
+              accessibilityLabel={`${I18n.t(
+                "wallet.ConfirmPayment.accessibility.transactionCosts",
+                {
+                  cost: formattedFees,
+                  psp: pspName
+                }
+              )}, ${I18n.t("wallet.ConfirmPayment.accessibility.edit")}`}
+            />
+
+            {isPayingWithPaypal && privacyUrl && (
               <>
                 <View spacer={true} />
-                <PayPalCheckoutPspComponent
-                  onEditPress={handleOnEditPaypalPsp}
-                  fee={fee as ImportoEuroCents}
-                  pspName={props.paypalSelectedPsp?.ragioneSociale ?? "-"}
-                  privacyUrl={props.paypalSelectedPsp?.privacyUrl}
-                />
+
+                <Text
```

Comment:
```suggestion
                <TouchableDefaultOpacity
```
Could it be an improvement, since this is used only as "tappable" container?